index.html

```diff
@@ -75,86 +79,15 @@ <h2 id="supporting-text">I am a Software Developer</h2>
-              sign-ups required. has been the industry's standard</p>
-            <ul class="other-content-tags">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <a class="other-content-link" href="#" target="_blank">See Project</a>
-          </div>
-          <div class="content-defaults content3">
-            <h2 class="other-content-title">Profesional Art Printing Data</h2>
-            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard</p>
-            <ul class="other-content-tags">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <a class="other-content-link" href="#" target="_blank">See Project</a>
-          </div>
-          <div class="content-defaults content4">
-            <h2 class="other-content-title">Profesional Art Printing Data</h2>
-            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard</p>
-            <ul class="other-content-tags">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <a class="other-content-link" href="#" target="_blank">See Project</a>
-          </div>
-          <div class="content-defaults content5">
-            <h2 class="other-content-title">Profesional Art Printing Data</h2>
-            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard</p>
-            <ul class="other-content-tags">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <a class="other-content-link" href="#" target="_blank">See Project</a>
-          </div>
-          <div class="content-defaults content6">
-            <h2 class="other-content-title">Profesional Art Printing Data</h2>
-            <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
-              sign-ups required. has been the industry's standard</p>
-            <ul class="other-content-tags">
-              <li>html</li>
-              <li>bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-            <a class="other-content-link" href="#" target="_blank">See Project</a>
-          </div>
-        </div>
+      <div class="works_content" id="worksContent">
+        <!-- div for class="featured-content"> -->
+        <div class="other-content" id="otherContent"></div>
       </div>
     </div>
 
+    <!-- works popup -->
+    <div class="works-popup" id="popupBox"></div>
+
```

Comment:
- [ ] Just as the previous reviewer pointed out, the issue still persists. You will notice that the popup is not centered from the bottom part. Kindly center the pop-up to make it look like the Figma design as posted by the previous reviewer.
![image](https://user-images.githubusercontent.com/86200356/202432357-4dfd1614-f82b-48e4-affc-97f0c480e642.png)